litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -695,6 +695,7 @@ spec:
       containers:
         - name: mongo
           image: litmuschaos/mongo:4.2.8
+          args: ["--ipv6"]
```

Comment:
Will it work for ipv4 with this ipv6 flag? @gdsoumya

Reply:
works for both